charts/seed-controlplane/charts/kube-apiserver/values.yaml

```diff
@@ -4,9 +4,12 @@ kubernetesVersion: 1.8.5
 advertiseAddress: 127.0.0.1
 securePort: 443
 probeCredentials: base64(user:pass)
-serviceNetwork: 10.0.0.0/24
-podNetwork: 192.168.0.0/16
-nodeNetwork: 172.16.0.0/20
+shootNetworks:
+  service: 10.0.0.0/24
+seedNetworks:
+  service: 172.16.0.0/20
+  pod: 10.0.0.0/24 
+  node: 192.168.0.0/16
```

Comment:
In the seedNetworks, it seems the IP ranges got rotated:
- service -> pod
- pod -> node
- node -> service

Is it the case? And if so why was it rotated?

Reply:
There is no specific reason behind rotation. I just blindly applied it. Actual values for these networks come from seed YAML while applying chart, so these value are just default placeholder. And there is no constraint over there network except those needs to be from private IP ranges.
Another point is, earlier network variables declared here were indicating shoot networks, not seed networks, so in that perspective there is no relation at all 😉